scripts/refresh-assets.ts

```diff
@@ -402,7 +402,7 @@ async function writeAssetDeclarationDTSFile() {
 }
 
 async function copyVersionToDotCom() {
-	const packageVersion = await import(join(REPO_ROOT, 'packages', 'tldraw', 'package.json')).then(
+	const packageVersion = await import('file://' + join(REPO_ROOT, 'packages', 'tldraw', 'package.json')).then(
 		(pkg) => pkg.version
 	)
```

Comment:
Rather than using `import` to load a JSON file like this, lets use our `readJsonIfExists` helper:

```suggestion
	const packageVersion = (await readJsonIfExists(join(REPO_ROOT, 'packages', 'tldraw', 'package.json'))).version
```